packages/hash/frontend/src/components/layout/PageSidebar/NavLink.tsx

```diff
@@ -32,20 +31,12 @@ export const NavLink: FC<NavLinkProps> = ({
       size="small"
       unpadded
       rounded
-      data-testid="create-page-btn"
+      className="end-adornment-btn"
```

Comment:
I’d prefer `-button` instead of `-btn` but we can leave this for now. Same for `evt`, `e`, `err` etc. Long names are shortened by code minification and compression. Short names confuse humans.

Reply:
Noted. will update